cli/build.go

```diff
@@ -15,6 +15,7 @@ import (
 	"path/filepath"
 	"runtime"
 	"strings"
+	"sync"
 	"time"
 
 	"github.com/johnewart/archiver"
```

Comment:
For reference, this is only because I have a patch that fixes tar files with hard links, which I think Flutter had an issue with

Reply:
Opened https://app.clubhouse.io/yourbaseio/story/2561/use-upstream-archiver-in-cli